tools/Cron_jobs/quarantine_report.php

```diff
@@ -219,7 +219,13 @@
      ishighspam=0
     AND
      ishighmcp=0";
-}  
+}
+
+if (defined('HIDE_NON_SPAM') && HIDE_NON_SPAM) {
```

Comment:
Done.  I will go back and update other options with this as well for consistency later.